Lib/importlib/_abc.py

```diff
@@ -1,11 +1,16 @@
 """Subset of importlib.abc used to reduce importlib.util imports."""
 from . import _bootstrap
 import abc
+import warnings
 
 
 class Loader(metaclass=abc.ABCMeta):
 
     """Abstract base class for import loaders."""
+    def __init__(self):
+        warnings.warn(f"Loader is deprecated.",
+                      DeprecationWarning, stacklevel=2)
+        super().__init__()
```

Comment:
I'm not 100% sure but I think we should raise the deprecation warning when the class is imported, not when you create a new instance with it. Going by the example in the original [PEP 562](https://peps.python.org/pep-0562/) it'd look (I think) something like this:

```python
# Rename the deprecated class
class _Loader(metaclass=abc.ABCMeta):
    ...

def __getattr__(name):
    if name == 'Loader':
        warnings.warn(f"The 'Loader' class is deprecated.",
                      DeprecationWarning, stacklevel=2)
        return _Loader
    raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
```

Reply:
Ok, no problem. Thanks for the reference link! Another approach I had was to just put the warning message by itself in the first line under the class name. Would this have a similar affect? I removed it because I then saw the deprecation error when running the ```make``` command.

Reply:
I think the problem with the other approach is that you'll get the warning whenever you import the module, not just the deprecated class specifically. For example, `import importlib.abc` will produce a warning for `InspectLoader` even if you're not actually using it